src/test/java/org/folio/support/MatchUtils.java

```diff
@@ -0,0 +1,65 @@
+package org.folio.support;
+
+import static org.hamcrest.Matchers.contains;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+
+import java.time.OffsetDateTime;
+import java.util.Collection;
+import java.util.LinkedHashMap;
+import java.util.Objects;
+import lombok.experimental.UtilityClass;
+import org.folio.entlinks.domain.dto.BibStatsDto;
+import org.folio.entlinks.domain.dto.BibStatsDtoCollection;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.springframework.test.web.servlet.ResultMatcher;
+
+@UtilityClass
+public class MatchUtils {
```

Comment:
Could you please also move match-methods from `IntegrationTestBase` to this class?